pkg/service/user.go

```diff
@@ -943,6 +943,8 @@ func (s *userService) RetrieveCliConfig(ctx context.Context, req *userrpcv3.ApiK
 		}
 	}
 
+	CreateApiKeyAuditEvent(ctx, s.al, AuditActionDownload, req.Username)
```

Comment:
we are not downloading ApiKey here, this should be present within above condition while creating.

if apikey == nil {
		apikey, err = s.ks.Create(ctx, &userrpcv3.ApiKeyRequest{Username: req.Username, Id: req.Id})
		if err != nil {
			return nil, err
		}
CreateApiKeyAuditEvent(ctx, s.al, AuditActionCreate, req.Username)
	}